index.html

```diff
@@ -3752,6 +3752,69 @@ <h3>Media Type Precision</h3>
           </p>
         </section>
 
+        <section>
+          <h2>JSON Processing</h2>
+
+          <p>
+While the base media types for this specification are JSON-LD documents, JSON-LD
+processing is not required to be performed in a variety of scenarios. These
+scenarios include, but are not limited to:
+          </p>
+
+          <ul>
+            <li>
+When using the JOSE or COSE specifications to secure a
+<a>verifiable credential</a> or <a>verifiable presentation</a>.
+            </li>
+            <li>
+When performing JSON Schema validation.
+            </li>
+            <li>
+When serializing or deserializing <a>verifiable credentials</a> or
+<a>verifiable presentations</a> into databases that index their contents.
+            </li>
+            <li>
+When operating on <a>verifiable credentials</a> or <a>verifiable
+presentations</a> in a software application after verification or validation
+is performed.
```

Comment:
```suggestion
When operating on <a>verifiable credentials</a> or <a>verifiable
presentations</a> in a software application after verification or validation
is performed for securing mechanisms that require an understanding of
and processing of JSON-LD.
```

Reply:
basically just want some language that calls out that for certain securing mechanisms, the JSON-LD is already processed and validated, therefore after those steps have happened, you are free to go on to just json.

Reply:
Yes, agreed, we'll get some variation of this in there, it's important to point it out.